core/src/components/checkbox/test/basic/checkbox.e2e.ts

```diff
@@ -5,57 +5,58 @@ test.describe('checkbox: basic', () => {
   test('should not have visual regressions', async ({ page }) => {
     await page.goto(`/src/components/checkbox/test/basic`);
 
-    await page.setIonViewport();
-
-    expect(await page.screenshot()).toMatchSnapshot(`checkbox-basic-${page.getSnapshotSettings()}.png`);
+    const content = page.locator('#checkboxes');
+    expect(await content.screenshot()).toMatchSnapshot(`checkbox-basic-${page.getSnapshotSettings()}.png`);
```

Comment:
Can we break these out into separate tests so it's clear what each checkbox is designed to test? For example, I have a `states` test for toggle where each screenshot tests a specific state: https://github.com/ionic-team/ionic-framework/blob/feature-7.0/core/src/components/toggle/test/states/toggle.e2e.ts